com.unity.visualeffectgraph/Editor/ShaderGraph/Templates/VFXConfigPlanarPrimitive.template.hlsl

```diff
@@ -8,7 +8,7 @@
 
 #define HAVE_VFX_PLANAR_PRIMITIVE
 
-bool GetMeshAndElementIndex(inout AttributesMesh input, inout AttributesElement element)
+bool GetMeshAndElementIndex(inout VFX_SRP_ATTRIBUTES input, inout AttributesElement element)
 {
     uint id = input.vertexID;
```

Comment:
This file has been modified by this PR: https://github.com/Unity-Technologies/Graphics/pull/5149
The changes should have been correctly reported.
See these [specific lines](https://github.com/Unity-Technologies/Graphics/pull/5149/files#diff-d60723623314f24c8b46851360b97e4a3453b1a724d740c51b37a629bb3916bdR76)